src/node/buffer.ts

```diff
@@ -0,0 +1,7 @@
+import * as bufferImpl from 'node-internal:bufferImpl';
```

Comment:
Nit: it's worth adding a comment here that this is a temporary placeholder ... it should be obvious but just to be clear